exampleSite/static/admin/config.yml

```diff
@@ -50,6 +50,70 @@ collections:
           name: "role"
           widget: "hidden"
           default: "healer"
+      - label: "Tanks"
+        name: "tanks"
+        file: "content/jobs/tanks/_index.md"
+        fields:
+        - {label: "Body", name: "body", widget: "markdown"}
+        - label: "Menu hierarchy"
+          name: "menu"
+          widget: "hidden"
+          default:
+            main:
+              name: "Tanks"
+              parent: "jobs"
+        - label: "Role"
+          name: "role"
+          widget: "hidden"
+          default: "tank"
+      - label: "Melees"
+        name: "melees"
+        file: "content/jobs/melees/_index.md"
+        fields:
+        - {label: "Body", name: "body", widget: "markdown"}
+        - label: "Menu hierarchy"
+          name: "menu"
+          widget: "hidden"
+          default:
+            main:
+              name: "Melees"
+              parent: "jobs"
+        - label: "Role"
+          name: "role"
+          widget: "hidden"
+          default: "melee"
+      - label: "Ranged"
+        name: "ranged"
+        file: "content/jobs/ranged/_index.md"
+        fields:
+        - {label: "Body", name: "body", widget: "markdown"}
+        - label: "Menu hierarchy"
+          name: "menu"
+          widget: "hidden"
+          default:
+            main:
+              name: "Ranged"
+              parent: "jobs"
+        - label: "Role"
+          name: "role"
+          widget: "hidden"
+          default: "ranged"
+      - label: "Casters"
+        name: "casters"
+        file: "content/jobs/casters/_index.md"
+        fields:
+        - {label: "Body", name: "body", widget: "markdown"}
+        - label: "Menu hierarchy"
+          name: "menu"
+          widget: "hidden"
+          default:
+            main:
+              name: "Casters"
+              parent: "jobs"
+        - label: "Role"
+          name: "role"
+          widget: "hidden"
+          default: "caster"
```

Comment:
Just to make 100% sure – you used the editor to generate the other `_index.md` files as part of this PR?

Reply:
Yes that is correct :)

Reply:
actually i'm confusing myself a bit with the Job landing page one. Scratch my previous comment. I didn't check it. Let me do that now.

Reply:
ugh these hidden labels again. Let me try to set them up so that they are visible

Reply:
They're tricky beasts. That's why I thought just going with selections with the value hardcoded might be best.

Reply:
Yep! i've turned them into selections in my recent updates. Feel free to review it again @nonowazu